framework/src/outputs/formatters/SONDefinitionFormatter.C

```diff
@@ -225,11 +233,29 @@ SONDefinitionFormatter::addBlock(const std::string & block_name,
 void
 SONDefinitionFormatter::addParameters(const JsonVal & params)
 {
+
   for (const auto & param_name : params.getMemberNames())
   {
 
     JsonVal param = params[param_name];
 
+    // lambda to calculate relative path from the current level to the document root
+    auto backtrack = [this]() -> std::string {
+      std::string backtrack_path;
+      for (size_t i = 0; i < _level; ++i)
```

Comment:
`size_t` should be an `int` since `_level` is an `int`

Reply:
@brianmoose -
Good catch - I update this and pushed it back.